tc/core/polyhedral/llvm_jit.cc

```diff
@@ -103,11 +104,21 @@ void Jit::addModule(std::shared_ptr<Module> M) {
   auto res = compileLayer_.addModule(M, std::move(Resolver));
   TC_CHECK(res) << "Failed to jit compile.";
 }
-#else
+
+#elif LLVM_VERSION_MAJOR >= 6
+
 Jit::Jit()
-    : SSP(),
+    :
+#if LLVM_VERSION_MAJOR == 6
```

Comment:
nested macro-ifs look unintuitive

Reply:
well it's actually wrong now that I see it in a diff ..